nullaway/src/main/java/com/uber/nullaway/handlers/OptionalEmptinessHandler.java

```diff
@@ -72,14 +72,14 @@ public boolean onOverrideMayBeNullExpr(
   public void onMatchTopLevelClass(
       NullAway analysis, ClassTree tree, VisitorState state, Symbol.ClassSymbol classSymbol) {
     optionalTypes =
-        config
-            .getOptionalClassPaths()
-            .stream()
-            .map(
-                type ->
-                    Optional.ofNullable(state.getTypeFromString(type))
-                        .map(state.getTypes()::erasure))
-            .collect(Collectors.toSet());
+        ImmutableSet.copyOf(
+            config
+                .getOptionalClassPaths()
+                .stream()
+                .map(state::getTypeFromString)
+                .filter(Objects::nonNull)
+                .map(state.getTypes()::erasure)
+                .collect(Collectors.toSet()));
```

Comment:
Guava 22 has collectors: https://google.github.io/guava/releases/22.0/api/docs/com/google/common/collect/ImmutableSet.html#toImmutableSet--

Reply:
So no need to build a Java set and call `copyOf(...)`

Reply:
Thanks! Done 👍